docs/usage.rst

```diff
@@ -334,7 +334,14 @@ Examples
     -rw-r--r-- root   root       1383 May 22 22:25 etc/ImageMagick-6/colors.xml
     ...
 
-
+    $ borg list /mnt/backup::root-2013-08-02 --list-format="{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NEWLINE}"
+    drwxr-xr-x root   root          0 Jun 05 12:06 .
```

Comment:
I took a short cut and just copied the example, because that's the same formatting as the print function produced.
If you have better name for "ISO-8601 date and time format" produced by format_time(t) I'd be happy to use that.